docs/guides/mnq_with_nats_terraform_provider.md

```diff
@@ -0,0 +1,110 @@
+---
+page_title: "Using Scaleway Messaging and Queuing service with NATS Terraform provider"
+description: |-
+Using Scaleway Messaging and Queuing service with NATS Terraform provider
+---
+
```

Comment:
General remark: I find it hard to follow the constant switching between terraform and nats cli